src/analytics.ts

```diff
@@ -12,12 +12,12 @@ export function createAnalyticsInstance(options?: Options) {
     let _growthbook: Growthbook,
         _rudderstack: RudderStack,
         core_data: Partial<TCoreAttributes> = {},
-        cta_buttons: Record<keyof TEvents, boolean> | {} = {},
-        offline_cache: any = {}
+        tracking_config: { [key: string]: boolean } = {},
+        offline_cache: { [key: string]: { event: keyof TEvents; payload: TEvents[keyof TEvents] } } = {}
 
     let interval = setInterval(() => {
```

Comment:
why having this interval is necessary? @NikitK-deriv

Reply:
in case that the growthbook is disabled from outside then this interval will run endlessly. 
I think we may need to come up with a better implementation for this.